CombinedDocument.md

```diff
@@ -1200,3 +1200,27 @@ Contract testing is used to verify that an API outside of your direct control is
 
 ### Blackbox Testing
 Blackbox testing is performed from the perspective of a user of the application, to verify that functionality the user expects to work does, in fact, work. Always perform blackbox testing on an actual device, to match the environment a user would experience as close as possible.
+
+# Zero Usage
+
+### How do we use `zero`?
+
+`zero` exists as a representation of the zero value of a given structure or primitive such as `CGPoint` or `Int`.
```

Comment:
Should we link any of the code highlighted items to the Swift docs for quick reference?

Reply:
No, I don't think so, not for this.